bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/LocaleProvider.java

```diff
@@ -25,4 +28,10 @@
      */
     Locale getLocale();
 
+    /**
+     * Provides access to the location of the installation
+     *
+     * @return location of the current installation or null if the location is not set
+     */
+    PointType getLocation();
```

Comment:
> If we find a good name

Hm, I18N seemed to be the very best choice, but this is already taken :-/

> But I don't think we should merge I18nProvider and the LocaleProvider because the I18nProvider offers functions to localize text. The LocaleProvider itself holds a configuration.

Hm, maybe we will also offer services around locale&location later on in that service, like e.g. formatting numbers/strings according to a locale, getting a local currency or whatever...
Why should a configuration be necessarily be separated from functionality?